learning_journal/views/default.py

```diff
@@ -25,8 +25,7 @@ def write(request):
         new_title = request.POST["title"]
```

Comment:
Can't comment above this line, but why do you have any code dealing with a DBAPIError if you never actually intend to raise it?